robottelo/ui/subscription.py

```diff
@@ -30,9 +31,12 @@ def upload(self, path, repo_url=None):
             self.text_field_update(locators['subs.repo_url_update'], repo_url)
             self.click(common_locators['save'])
         browse_element = self.wait_until_element(locators['subs.file_path'])
-        browse_element.send_keys(path)
+        with open(manifest.filename, 'wb') as handler:
```

Comment:
I'm not sure I follow this... are we taking a file-like object and writing it to disk locally (to the Satellite server)? If so, maybe comment here?